torch/_dynamo/output_graph.py

```diff
@@ -404,6 +404,11 @@ def __init__(
 
         self.guard_on_key_order: Set[str] = set()
 
+        # Track compiled autograd final callbacks that must be called at the end of this graph.
+        # Only applicable if this graph is created from Dynamo tracing in Compiled Autograd.
+        self.ca_final_callbacks: List[Callable] = []
```

Comment:
Since dynamo doesn't actually mutate variables while tracing this isn't needed, it will always remain empty.  You can just pass in `[]` below.